tests/command.dev.test.js

```diff
@@ -0,0 +1,793 @@
+const test = require('ava')
```

Comment:
This file was refactored and renamed from `dev.test.js`.

Each test creates a dedicated site with the relevant settings just for that test